api/envoy/config/core/v3/protocol.proto

```diff
@@ -26,7 +26,33 @@ message TcpProtocolOptions {
       "envoy.api.v2.core.TcpProtocolOptions";
 }
 
+//
+message QuicKeepAliveSettings {
+  // The max interval in seconds for a connection to send keep-alive probing packets
+  // (with PING or PATH_RESPONSE).
+  //
+  // If initial_interval is zero, a client connection will use this value to
+  // start probing. In such case, the value should be smaller than
+  // :ref:`connection idle_timeout <envoy_v3_api_field_config.listener.v3.QuicProtocolOptions.idle_timeout>` to
+  // prevent idle timeout while not too small to throttle the connection or to flood the peer.
+  // If zero, disable keepalive probing.
+  google.protobuf.UInt32Value max_interval = 1 [(validate.rules).message = {required: true}];
+
+  // The interval in seconds to send the first 6 keep-alive probing packet to prevent
+  // connection from idle timeout. The following probing will be sent, each
+  // one with an interval exponentially longer than previous one, till it reaches
+  // max_interval. And the probing afterwards will always use max_interval.
+  //
+  // The value should be smaller than :ref:`connection idle_timeout <envoy_v3_api_field_config.listener.v3.QuicProtocolOptions.idle_timeout>` to
```

Comment:
This line seem particularly long. Is it wrapped by the format fixer?

Reply:
The formatter keeps it as it's in a :ref: link.